src/loaders/scraper_browse_id.py

```diff
@@ -15,10 +16,14 @@ def save_browse_page(content: str, page: int) -> None:
         page (int): page number to identify file with
     '''
     filename = f'{config.DATA_PATH}/raw/bgg_browse_page_{str(page)}.html'
+    pathname = f'{config.DATA_PATH}/raw/'
     try:
         with open(filename, 'xb') as file:
             file.write(content)
-    except FileExistsError:
+    except (FileExistsError, FileNotFoundError):
```

Comment:
These two exceptions need to be in separate code blocks (use a second except clause for the new FileNotFoundError). If the file does not exist (which is implied by the lack of a directory), then we need to use the 'xb' permissions in open() to create the file, instead of 'wb' which writes to an existing file.

Great catch!

Reply:
I think we can use here only 'wb' permission , it will overwrite existing file and create a new file ,and check only for 'FileNotFoundError'

Reply:
I like that idea, much cleaner!